### PR TITLE
Use rst instead of md for a rst file

### PR DESCRIPTION
**Pull request by muffinresearch**

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -4,11 +4,23 @@
...
+-  Follow `PEP8`_, `jshint`_ and our other `style guide conventions`_.
+-  Please write tests and read the docs on `addons-server`_.
 
 And thank you for contributing code.
+
+.. _license: https://github.com/mozilla/addons-server/blob/master/LICENSE
+.. _dev-addons mailing list: https://mail.mozilla.org/listinfo/dev-addons
+.. _PEP8: http://www.python.org/dev/peps/pep-0008/
+.. _jshint: http://www.jshint.com/
+.. _style guide conventions: http://mozweb.readthedocs.org/en/latest/index.%20html
```
Comment (wagnerand) [inline CONTRIBUTING.rst]: Accidental `%20`?

Comment (muffinresearch) [inline CONTRIBUTING.rst]: Hmm, probably came via pandoc. Good catch.

**Comment by wagnerand** [comment]
Can you make all links except _jshint_ HTTPS?

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -4,11 +4,23 @@
...
+then please file an issue on the `mozilla/addons`_ project instead.
+
 Before contributing code, please note:
 
-* You agree to license your contributions under the [license](https://github.com/mozilla/addons-server/blob/master/LICENSE).
-* Ask on the [dev-addons mailing list](https://mail.mozilla.org/listinfo/dev-addons) or in GitHub before making changes.
-* Follow [PEP8](http://www.python.org/dev/peps/pep-0008/), [jshint](http://www.jshint.com/) and our other [style guide conventions](http://mozweb.readthedocs.org/en/latest/index.html).
-* Please write tests and read the docs on [addons-server](http://addons-server.readthedocs.org/en/latest/).
+-  You agree to license your contributions under the `license`_.
+-  Ask on the `dev-addons mailing list`_ or in GitHub before making
```
Comment (wagnerand) [inline CONTRIBUTING.rst]: Not sure what exactly `in GitHub` refers to? Let's clarify? (I know that didn't change, but while we're at it...)

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -4,11 +4,23 @@
...
+   changes.
+-  Follow `PEP8`_, `jshint`_ and our other `style guide conventions`_.
+-  Please write tests and read the docs on `addons-server`_.
 
 And thank you for contributing code.
```
Comment (wagnerand) [inline CONTRIBUTING.rst]: I would delete `code` here. Makes it more welcome, as people could contribute more than just code (e.g. images).

**Comment by wagnerand** [comment]
r+wc

**Pull request merged**